portfolio/src/main/webapp/index.html

```diff
@@ -142,13 +143,37 @@ <h1>malachi blenman
                 </div>
             </div>
         </section>
+<!-- ZODIAC --> 
+        <section class="zodiac" id="zodiac">
+        <h1>choose your astrological sign!</h1>
+        <form action="/zodiacsign" method="POST">
+        <select name="astrosign">
+            <option value="Aries">Aries</option>
+            <option value="Taurus">Taurus</option>
+            <option value="Gemini">Gemini</option>
+            <option value="Cancer">Cancer</option>
+            <option value="Leo">Leo</option>
+            <option value="Virgo">Virgo</option>
+            <option value="Libra">Libra</option>
+            <option value="Scorpio">Scorpio</option>
+            <option value="Sagittarius">Sagittarius</option>
+            <option value="Capricorn">Capricorn</option>
+            <option value="Aquarius">Aquarius</option>
+            <option value="Pisces">Pisces</option>
+        </select>
+        <button onclick = "drawChart()">Submit</button>
```

Comment:
Does the data get submitted properly? I thought you might need an `<input type="submit">` here to actually send the request to the server. I could be mistaken though.